src/api_routes_http.rs

```diff
@@ -320,7 +321,8 @@ pub fn config(cfg: &mut web::ServiceConfig, rate_limit: &RateLimitCell) {
           .route("/totp/generate", web::post().to(generate_totp_secret))
           .route("/totp/update", web::post().to(update_totp))
           .route("/list_logins", web::get().to(list_logins))
-          .route("/validate_auth", web::get().to(validate_auth)),
+          .route("/validate_auth", web::get().to(validate_auth))
+          .route("/list_media", web::get().to(list_media)),
```

Comment:
Like the comment above says, I would already put this under `/account` so its one less thing we need to migrate later.

Reply:
I don't think we should do this prematurely, it should be all at once, with the rest of these as well. The `/account` heading doesn't exist yet anyway.

Reply:
I agree with @Nutomic because changing less endpoints later means less changes to client code.

Reply:
Only this still missing, otherwise looks good to merge.

Reply:
Still needed.

Reply:
Done